OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -22,11 +22,10 @@ public interface Analytics {
      *
```

Comment:
Please resolve warnings

Reply:
We still have a method `trackInAppPurchasesEvent` which is using `action` as a key

Reply:
There are over 50 warnings in this file, but they are not related to this PR. Plus as mentioned in the PR description, we will ignore this key for the IAP events ([decision](https://2u-internal.atlassian.net/browse/LEARNER-9973?focusedCommentId=4952957)). 👍🏻